docs/en/connector-v2/sink/Paimon.md

```diff
@@ -31,7 +31,7 @@ libfb303-xxx.jar
 
```

Comment:
Piamon version 0.7 is incompatible with version 0.9.  You are advised to provide the mapping between the ST Paimon Connector and the Paimon version.